projects/canopy/src/lib/modal/modal-body-timer/modal-body-timer.component.html

```diff
@@ -1 +1 @@
-{{ this.timer }}
+{{ timer }}
```

Comment:
Also wondering if this should be a separate commit, as it doesn't look related to the unit test changes.

Reply:
Placed this in a separate commit. See: 7e17a3ab16d28a24086f4d1e770a7ac65c7e097b